### PR TITLE
Add gomod.sum, extend .gitignore, add integrations

### PR DESCRIPTION
**Pull request by dtitov**
- Add missing gomod.sum
- Extend .gitignore
- Add GitHub Actions (empty test run for now)
- Add a couple of integrations with quality-check services
- Add badges for the above services to the README.md
- go fmt

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,10 @@
+# Created by https://www.gitignore.io/api/go
+# Edit at https://www.gitignore.io/?templates=go
+
+### Go ###
 # Binaries for programs and plugins
+/dist
+lega-uploader
```
Comment (jbygdell) [inline .gitignore:7]: copy paste error?

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  # Keep go.mod (& go.sum) up to date, batching pull requests daily
+  - package_manager: "go:modules"
+    directory: "/"
+    update_schedule: "daily"
+  # Keep Dockerfile up to date, batching pull requests weekly
+  - package_manager: "docker"
+    directory: "/"
+    update_schedule: "weekly"
```
Comment (jbygdell) [inline .dependabot/config.yml:10]: no new line at end of file

File: .gitignore [hunk 3]
```diff
@@ -13,3 +19,11 @@
...
 # Dependency directories (remove the comment below to include it)
 # vendor/
+
+### Go Patch ###
+/vendor/
+/Godeps/
+
+# End of https://www.gitignore.io/api/go
+
+coverage.txt
```
Comment (jbygdell) [inline .gitignore:29]: no new line at end of file

File: .gitignore (hunk 1 above)
Comment (dtitov) [reply .gitignore:7]: Ooops, yes.

**Pull request merged**